Source/MVC5/Boilerplate.Web.Mvc5/Twitter/Cards/PlayerTwitterCard.cs

```diff
@@ -4,21 +4,25 @@
     using System.Text;
 
     /// <summary>
-    /// Video clips and audio streams have a special place on the Twitter platform thanks to the Player Card. By implementing a few HTML meta tags to
-    /// your website and following the Twitter Rules of the Road, you can deliver your rich media to users across the globe.
-    /// Twitter must approve the use of the player card, find out more below.
+    /// Video clips and audio streams have a special place on the Twitter platform thanks to the Player Card. By
+    /// implementing a few HTML meta tags to your website and following the Twitter Rules of the Road, you can deliver
+    /// your rich media to users across the globe. Twitter must approve the use of the player card, find out more below.
```

Comment:
widescreen is offset by multiple parallel windows. I'll look at turning off the wrap entirely.  Personally. I prefer it not wrap.